app/jest.config.js

```diff
@@ -0,0 +1,19 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```

Comment:
Is this file auto-generated? Any reason not to pull in https://github.com/navapbc/frontend/blob/main/examples/nextjs-uswds-and-cmsds/jest.config.js?

Reply:
I did `config:init` -- but that only generates a file containing the following: 

```// jest.config.js
/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
module.exports = {
  preset: "ts-jest",
  testEnvironment: "node",
}
``` 

I added everything else in this file to make our specific use-case work. I'm happy to add the set up options that are in that file and aren't already included -- i.e. `testRegex` and `testPathIgnorePatterns`